src/opm/output/eclipse/AggregateActionxData.cpp

```diff
@@ -255,7 +255,10 @@ const std::map<cmp_enum, int> cmpToIndex = {
             int max_l_str = 128;
             // write out the schedule input lines
             const auto& schedule_data = actx.keyword_strings();
-            for (auto z_data : schedule_data) {
+            for (auto zd : schedule_data) {
+                //remove leading blanks
+                std::size_t start = zd.find_first_not_of(" ");
+                const auto z_data = (start != zd.size()) ?  zd.substr(start) : zd;
```

Comment:
The `find_first_not_of()` will *never* return `zd.size()`, if the string is all blank you will rather get the return value `std::string::npos`. The comment says *remove leading blanks*  - that can be achieved with:
```C++
#include <common/utility/String.hpp>

const auto& z_data = ltrim_copy(zd);
```